src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreRequest.cs

```diff
@@ -38,7 +38,7 @@ public class RestoreRequest
             RestoreCommandProviders dependencyProviders,
             SourceCacheContext cacheContext,
             ClientPolicyContext clientPolicyContext,
-            PackageNamespacesConfiguration packageNamespaces,
+            PackageSourceMappingConfiguration packageSourceMapping,
```

Comment:
Considering previously we used plural to convey it's multiple namespace, should we use `packageSourceMappings` too?

Reply:
I think `mapping` is the right word. 

The definition of mapping itself refers to an NxM relationship, so the plural is unnecessary imo